statsmodels/iolib/summary2.py

```diff
@@ -360,7 +360,10 @@ def summary_params(results, yname=None, xname=None, alpha=.05, use_t=True,
                         '[' + str(alpha/2), str(1-alpha/2) + ']']
 
     if not xname:
-        data.index = results.model.exog_names
+        try:
+            data.index = results.model.data.param_names
+        except AttributeError:
+            data.index = results.model.exog_names
```

Comment:
`try except` was necessary because without it 3324 had a test error in MICE.
A proper solution needs to wait. I could have added a quickfix in MICE instead but overall this needs more thinking about the design.